### ISSUE TITLE
Error on image-upload

### ISSUE THREAD

**Issue by cleeyv** [linode/linode-cli#262]
When trying to use `linode-cli image-upload` I'm getting the following error on an Arch-based system with the build of linode-cli from the AUR (version 5.8.1):
```
$ linode-cli image-upload --label "nixos-test" nixos.img.gz
Traceback (most recent call last):
  File "/usr/bin/linode-cli", line 33, in <module>
    sys.exit(load_entry_point('linode-cli===fatal.-.N.o', 'console_scripts', 'linode-cli')())
  File "/usr/lib/python3.9/site-packages/linodecli/__init__.py", line 407, in main
    plugins.invoke(parsed.command, plugin_args, context)
  File "/usr/lib/python3.9/site-packages/linodecli/plugins/__init__.py", line 33, in invoke
    plugin.call(args, context)
  File "/usr/lib/python3.9/site-packages/linodecli/plugins/image-upload.py", line 85, in call
    context.client.config.update(parsed)
TypeError: update() missing 1 required positional argument: 'allowed_defaults'
```

I'm also seeing the same error when running linode-cli from the NixOS unstable channel (also version 5.8.1):
```
$  linode-cli image-upload --label "nixos-test" nixos.img.gz
Traceback (most recent call last):
  File "/nix/store/hhfxpxpvwavg85nnsihmdd09xpbqj8c2-linode-cli-5.8.1/bin/.linode-cli-wrapped", line 9, in <module>
    sys.exit(main())
  File "/nix/store/hhfxpxpvwavg85nnsihmdd09xpbqj8c2-linode-cli-5.8.1/lib/python3.9/site-packages/linodecli/__init__.py", line 407, in main
    plugins.invoke(parsed.command, plugin_args, context)
  File "/nix/store/hhfxpxpvwavg85nnsihmdd09xpbqj8c2-linode-cli-5.8.1/lib/python3.9/site-packages/linodecli/plugins/__init__.py", line 33, in invoke
    plugin.call(args, context)
  File "/nix/store/hhfxpxpvwavg85nnsihmdd09xpbqj8c2-linode-cli-5.8.1/lib/python3.9/site-packages/linodecli/plugins/image-upload.py", line 85, in call
    context.client.config.update(parsed)
TypeError: update() missing 1 required positional argument: 'allowed_defaults'
```

I have never used the image-upload plugin before so I don't have an example of a previously working version to compare to. Let me know if there's anything further I should try to workaround or debug this. Thanks.

**Comment by Dorthu** [comment]
Thanks for the report!  We only officially support the python distribution of the Linode CLI - the official method of installation is `pip install linode-cli`.  As such, I'd recommend trying the official distribution to see if that works (you can install it in a python virtualenv if you want to try it without interfering with your current installation).

That said, I'm going to turn up a new Arch machine and see if I can reproduce this issue.  I'll report back in a bit.

**Comment by cleeyv** [comment]
I have reproduced the error when running the pip-installed version of linode-cli (5.8.1) in a venv:
```
$ linode-cli image-upload --label "nixos-test" ../nixos.img.gz
Traceback (most recent call last):
  File "~/dev/linode-cli/env/bin/linode-cli", line 8, in <module>
    sys.exit(main())
  File "~/dev/linode-cli/env/lib/python3.9/site-packages/linodecli/__init__.py", line 407, in main
    plugins.invoke(parsed.command, plugin_args, context)
  File "~/dev/linode-cli/env/lib/python3.9/site-packages/linodecli/plugins/__init__.py", line 33, in invoke
    plugin.call(args, context)
  File "~/dev/linode-cli/env/lib/python3.9/site-packages/linodecli/plugins/image-upload.py", line 85, in call
    context.client.config.update(parsed)
TypeError: update() missing 1 required positional argument: 'allowed_defaults'
```

**Comment by Dorthu** [comment]
Thanks for verifying; I saw that too.  Pushed up a fix - my bad.  I'll get this released this afternoon.

**Comment by Dorthu** [comment]
This is fixed in version 5.8.2, which is live in pypi (I don't know when other repositories will update)
